src/main/controller/ExerciseController.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response } from 'express';
+import db from '../models';
+
+export class ExerciseController {
+  getAllExercises = async (_req: Request, res: Response) => {
+    try {
+      const exercises = await db.Exercise.findAll();
+      return res.status(200).send(exercises);
+    } catch (e) {
+      return res.status(400).send({ e });
```

Comment:
We'll probably want to consider adding an error handling class or a standard format so everything returns and propagates nicely & consistently! Do you want to get that setup?